go/vt/vtgate/plugin_mysql_server.go

```diff
@@ -64,6 +64,8 @@ var (
 	mysqlSslKey  = flag.String("mysql_server_ssl_key", "", "Path to ssl key for mysql server plugin SSL")
 	mysqlSslCa   = flag.String("mysql_server_ssl_ca", "", "Path to ssl CA for mysql server plugin SSL. If specified, server will require and validate client certs.")
 
+	mysqlSslServerCA = flag.String("mysql_server_ssl_server_ca", "", "combine server cert and server root CA, return full chain to client")
```

Comment:
Similar to the grpc flag, a bit more description is needed to clarify what should be provided as this flag value.

Reply:
thank you, updated phab, add more descriptive sentence